client/keystore/src/local.rs

```diff
@@ -501,6 +503,8 @@ mod tests {
 		str::FromStr,
 	};
 
+	const TEST_KEY_TYPE: KeyTypeId = KeyTypeId(*b"test");
```

Comment:
Nitpick: i prefer constants to be declared at the top instead of in-between implementations

Reply:
This is not between any implementations. It is at the top of the tests module.